stage1.js

```diff
@@ -0,0 +1,233 @@
+const myStreet = 'Pushkina'
+const numBuilding = 65
+const isAdress = true
+
+// Выполнить сложение различных типов(string+boolean, string+number, number+boolean) 
+
+console.log(myStreet + isAdress)
+console.log(myStreet + numBuilding)
+console.log(numBuilding + isAdress)
+
+//Выполнить умножение различных типов(string * boolean, string * number, number * boolean)
+
+console.log(myStreet * isAdress)
+console.log(myStreet * numBuilding)
+console.log(numBuilding * isAdress)
+
+//Выполнить деление различных типов(string/boolean, string/number, number/Boolean)
+
+console.log(myStreet / isAdress)
+console.log(myStreet / numBuilding)
+console.log(numBuilding / isAdress)
+
+//На склад принимают груши и яблоки, напишите функцию, которая возвращает результат сложения
+// количества принятых груш и яблок
+//1:
+function sumFruits(apples, pear) {
+  return apples + pear
+}
+const takenFruits = sumFruits(50, 30)
+console.log(takenFruits);
+//2:
+const FruitsSum = (яблоки, груши) => яблоки + груши
+console.log(FruitsSum(50, 30));
+
+//В терминале оплаты сохранено ваше имя, напишите функцию для определения имени
+// в терминале(если вы ввели ваше имя, то привет + имя, если нет, то нет такого имени)
+
+function login(name) {
+  const myName = 'Inna'
+  if (myName === name) {
+    console.log('Привет', name);
+  } else {
+    console.log('Нет такого имeни')
+  }
+}
+login('Inna')
+
+//Создать объект car, добавить к нему свойство color со значением 'черный'
+//Изменить свойство color объекта car на 'зеленый'
+//В объект car добавить свойство power, которое является функцией и 
+//выводит в консоль мощность двигателя
+
+const car = {
+  color: 'black',
+  power: function () {
+    console.log('engine power');
+  }
+}
+car.color = 'green'
+console.log(car);//{ color: 'green', power: [Function: power] }
+car.power()
+
+//Напишите функцию вычисления типа аргумента и вывод типа в консоль
+
+function showType(argument) {
+  console.log(typeof argument);
+}
+showType(6)
+
+// Напишите функцию, которая определяет является ли число простым или нет
+
+function isPrime(params) {
+  for (let i = 2; i < params; i++) {
```

Comment:
Doesn't work for numbers less than 2. All number less than 2 are not prime and it should be checked by your fuction.